README.md

```diff
@@ -37,6 +37,13 @@ Provide the SSH key for access to Dokku in the `SSH_PRIVATE_KEY` secret.
 
 `Procfile` contains the command for running the main process of the application.
 
+## Environment variables
+
+| Variable             | Default | Meaning                                                   |
+|----------------------|---------|-----------------------------------------------------------|
+| MAX_ATTEMPTS         | 4       | Number of attempts to restart a specific job.             |
```

Comment:
I think it should be `Number of attempts to run a specific workflow job.` after all.

Reply:
Done